### PR TITLE
Update kr.m3u

### PR DESCRIPTION
**Pull request by Alstruit**
Original Reference #9033:
Closed for a more clean history.

To implement:
Convert seeZN links to public links.
Append seeZN links 1080p by @LaneSh4d0w  scrapper tool.

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
you'll need the channel number over Seezn's data.
285 is Mnet
273 is JTBC
213 is AsiaN

**Comment by Alstruit** [comment]
Some channels from Seezn can be temporarly blocked at any time. Not sure if the 5 minute check is consistent.

**Comment by LaneSh4d0w** [comment]
> Some channels from Seezn can be temporarly blocked at any time. Not sure if the 5 minute check is consistent.

I think i'll need to put my link inside the playlist to recreate a m3u file for each session. This would prevent the program cutting off unexpectedly.

File: streams/kr.m3u [hunk 1]
```diff
@@ -325,3 +279,67 @@ http://necgokr2-724.acs.wecandeo.com/ms/2528/724/index.m3u8
...
+#EXTINF:-1 tvg-id="NationalAssemblyTV.kr" status="",National Assembly TV (720p)
+https://m.webcast.go.kr/live/_definst_/smil:natv_720p.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="NationalAssemblyTV.kr" status="",National Assembly TV (1080p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCL-WOj1FxKR8Hlzg5tvnWKg/live
+#EXTINF:-1 tvg-id="NHTV.kr" status="",NHTV (720p)
+http://nonghyup.flive.skcdn.com/nonghyup/_definst_/nhlive/playlist.m3u8
+#EXTINF:-1 tvg-id="ArtTVArte.kr" status="",Arte TV (1080p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCPglj7zGwmOlH8_31Tpgh8Q/live
+#EXTINF:-1 tvg-id="BravoKids.kr" status="",Bravo Kids (1080p)
+https://www.youtube.com/channel/UCu6mog947IWOXL23AzKlImw/live
```
Comment (freearhey) [inline streams/kr.m3u:309]:
```suggestion
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCu6mog947IWOXL23AzKlImw/live
```

File: streams/kr.m3u [hunk 2]
```diff
@@ -325,3 +279,67 @@ http://necgokr2-724.acs.wecandeo.com/ms/2528/724/index.m3u8
...
+#EXTINF:-1 tvg-id="HomeShopping.kr" status="",Home & Shopping (720p)
+https://s28.qtcdn.co.kr/media/liveM3U8/idx/518045142/enc/1860408350/playlist.m3u8
+#EXTINF:-1 tvg-id="HyundaiHomeShopping.kr" status="",Hyundai Home Shopping (720p)
+https://livejj.hyundaihmall.com:8443/live/ngrp:hmall.stream_pc/playlist.m3u8
+#EXTINF:-1 tvg-id="HyundaiHomeShoppingPlusShop.kr" status="",Hyundai Home Shopping Plus (720p)
+https://dtvstreaming.hyundaihmall.com/newcjp3/_definst_/newcjpstream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="LotteHomeShopping.kr" status="",Lotte Home Shopping (720p)
+https://pchlslivesw.lotteimall.com/live/livestream/lotteimalllive_mp4.m3u8
+#EXTINF:-1 tvg-id="" status="",Maeil Business Newspaper TV (매일경제 TV) (1080p)
+https://www.youtube.com/channel/UCnfwIKyFYRuqZzzKBDt6JOA/live
```
Comment (freearhey) [inline streams/kr.m3u:325]:
```suggestion
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCnfwIKyFYRuqZzzKBDt6JOA/live
```

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/kr.m3u [hunk 3]
```diff
@@ -113,30 +109,26 @@ http://mgugaklive.nowcdn.co.kr/gugakvideo/gugakvideo.stream/playlist.m3u8
...
-https://allanf181.github.io/adaptive-streams/streams/kr/JTBCGolf.m3u8
-#EXTINF:-1 tvg-id="JTBCGolfSports.kr" status="error",JTBC Golf & Sports (1080p) [Not 24/7]
-https://allanf181.github.io/adaptive-streams/streams/kr/JTBCGolfSports.m3u8
+#EXTINF:-1 tvg-id="JTBC2.kr" status="",JTBC2 (1080p)
+https://kr-streams.onrender.com/scraper?channel=455
+#EXTINF:-1 tvg-id="JTBC4.kr" status="",JTBC4 (1080p)
+https://kr-streams.onrender.com/scraper?channel=523
+#EXTINF:-1 tvg-id="JTBCGolf.kr" status="",JTBC Golf (1080p)
+https://kr-streams.onrender.com/scraper?channel=299
+#EXTINF:-1 tvg-id="JTBCGolfSports.kr" status="",JTBC Golf & Sports (480p) [Not 24/7] [Geo-locked]
```
Comment (LaneSh4d0w) [inline streams/kr.m3u:126]:
```suggestion
#EXTINF:-1 tvg-id="JTBCGolfSports.kr" status="",JTBC Golf & Sports (480p) [Not 24/7] [Geo-blocked]
```

**Review by freearhey — DISMISSED** [review]

**Review by MapGuy11 — DISMISSED** [review]

**Review by what-a — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
``

**Review by what-a — APPROVED** [review]

**Pull request merged**